docs/rules/text-area-missing-label.md

```diff
@@ -4,6 +4,10 @@
 
 <!-- end auto-generated rule header -->
 
+💼 This rule is enabled in the ✅ `recommended` config.
```

Comment:
I think this is duplicated code from running the readme update file and we don't need the changes.

Reply:
this was auto generated on running the docs-update command - will check later.

Reply:
I think there is a problem with the docs-update plugin, it always adds extra things at the top.